### PR TITLE
UX fixes

### LINKED ISSUE #75: Improve the UX of the "views" and "like" chips
In the story preview sheet, two chips are displayed containing the number of views and number of likes. When signed in, the likes chip becomes greyed out but also clickable. The following improvements are to be introduced:
- [ ] Allow tapping the like button even when not signed in (but perhaps redirect to the login screen or show a toast message)
- [ ] Change the colors to indicate the "views" is not clickable (maybe transparent), and to make it clearer when "likes" is toggled or not (perhaps white/blue by default, red when liked?)

### LINKED ISSUE #81: Allow changing the user's name in the profile
When logging in via email, the user's name is displayed as "`[no name]`" with no opportunity to fix it. Users should be allowed to change their `displayName`.
One easy fix is to bring back the Profile page screen from FlutterFire UI

### LINKED ISSUE #83: Rework the "pause" menu in the story playback
In the story play screen, there is currently a floating action button on the top right with a hamburger icon that, when tapped, opens a menu dialog with the options to "resume", "restart", and "exit". The way the menu open in a dialog and the first option saying "resume" imply that this button is a "pause" button, yet it has a hamburger icon rather than a pause ⏸ one.
This should either be changed to a pause icon ⏸︎, or trigger a floating menu (right below the button) without the option to "resume" (instead dismissing the menu simply by tapping outside it)

### LINKED ISSUE #85: Give the typing indicator a message shape
The typing "..." indicator looks more like a loading indicator with the way it appears in the middle of the screen, not associated with any actor. It should be moved to the side and appear as the content of the message to resemble instant messaging platforms (ex: Facebook Messenger) people are already used to.

### LINKED ISSUE #88: Use more accurate action messages in dialogs
In the play screen, when pausing and clicking on the "Restart" button, a dialog pops up whose action buttons say "Leave" and "Stay". Clicking "leave" doesn't actually leave, and it could be confusing that both words are not related to _restart_ing.
Change the buttons to say "Restart story" (confirmation action) and "cancel"

### PR DESCRIPTION
**Pull request by alkaitagi**
- [x] Closes #83 
- [x] Closes #85 
- [x] Closes #88
- [x] Remove redundant `actor.type` property and all associated UI elements
- [x] Closes #81  
- [x] Show only the first tag on `StoryTile` (little space)
- [x] Closes #75

### CODE REVIEW

**Comment by alkaitagi** [comment]
One bug that needs to be resolved is properly stopping the time. We should somehow watch for any route change, i.e. when any dialog, popup or other modal opens, and control timers on those events.

**Comment by alkaitagi** [comment]
Another issue I'm having is that `Profile`'s `setState` is being correctly fired after user renaming, but the actual UI label doesn't update its text.

File: lib/modules/profile/screens/profile.dart [hunk 1]
```diff
@@ -28,6 +29,10 @@ class _ProfileScreenState extends State<ProfileScreen> {
   @override
   void initState() {
     super.initState();
+    FirebaseAuth.instance.userChanges().listen((event) => setState(() {
+          print('CHANGED');
+          // this fires but the UI doesn't update
+        }));
```
[the remark this answers is not in the document]
Comment (aabounegm) [reply lib/modules/profile/screens/profile.dart:35]: Another thing: by printing `user.displayName`, it seems that it still has the old value, while the new value is in `event.displayName` (`event` should technically be `newUser`).

**Review by aabounegm — CHANGES_REQUESTED** [review]
see annotations by CI

**Review by aabounegm — APPROVED** [review]

**Comment by aabounegm** [comment]
Hmm, looks like the merge from master didn't happen as it should have

**Pull request merged**